### PR TITLE
replace with term of "samping frequency"

### PR DESCRIPTION
**Pull request by ytakio**
`sampling frequency` is technical term. It may not confuse reader with a correct term.

Issue #, if available:

Description of changes:

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by gadams999** [comment]
Thanks for the submission and concur on changes. I did notice another "sample" in the file (started a review). If you wouldn't mind changing that I'll then run through actions and upon validation push.

**Comment by ytakio** [comment]
@gadams999 Thank you for your confirmation!
Of course, I don't mind if you did that :)

**Review by gadams999 — APPROVED** [review]

File: src/hugo/content/en-us/patterns/telemetry/_index.md [hunk 1]
```diff
@@ -58,11 +58,11 @@ If large messages **can be sent in batches**, each message should be saved as a
 
 #### What are the sample vs. reporting frequencies of a device?
```
[the remark this answers is not in the document]
Comment (ytakio) [reply src/hugo/content/en-us/patterns/telemetry/_index.md]: Thanks! It looks great  to me 👍

**Comment by ytakio** [comment]
So sorry, I mistakedn that I rebased....
I'd appreciate if you would re approved 🙇 💦

**Pull request merged**